### PR TITLE
[Docs] Update Japanese translation of cli.md

### PR DESCRIPTION
**Pull request by shelaf**
## Description

Update Japanese translation of cli.md

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by mtei** [comment]
@umi-umi @s-show @hsgw よかったらレビューお願いします。

File: docs/ja/cli.md [hunk 1]
```diff
@@ -1,29 +1,19 @@
...
-* [ローカル CLI](#local-cli)
-* [CLI コマンド](#cli-commands)
-
-# 必要事項
-
-CLI は Python 3.5 以上を必要とします。我々は必要事項の数を少なくしようとしていますが、[`requirements.txt`](https://github.com/qmk/qmk_firmware/blob/master/requirements.txt) にリストされているパッケージもインストールする必要があります。
+### 必要事項 :id=requirements
 
-# グローバル CLI :id=global-cli
+CLI は Python 3.5 以上を必要とします。我々は必要事項の数を少なくしようとしていますが、[`requirements.txt`](https://github.com/qmk/qmk_firmware/blob/master/requirements.txt) にリストされているパッケージもインストールする必要があります。これらは QMK CLI をインストールするときに自動的にインストールされます。
```
Comment (s-show) [inline docs/ja/cli.md]: 「リストされている」より「列挙されている」の方が良いかなと思いますが、好みの問題でもありますので、Approve します。修正するか否かはお任せします。
```suggestion
CLI は Python 3.5 以上を必要とします。我々は必要事項の数を少なくしようとしていますが、[`requirements.txt`](https://github.com/qmk/qmk_firmware/blob/master/requirements.txt) に列挙されているパッケージもインストールする必要があります。これらは QMK CLI をインストールするときに自動的にインストールされます。
```

**Comment by mtei** [comment]
I think this is ready to merge.

**Comment by mtei** [comment]
Thanks !

**Pull request merged**